### DISCUSSION TITLE
Get default laravel User model into Twill

### DISCUSSION THREAD

**Discussion by manuelsofia** [area17/twill#898] [Q&A] [answered]
Hi there... I need a section to list all the Users (the laravel default User model, not the twill admin users) to access them and edit them through Twill. How can I achieve that? I know that for any other model I can extend the Twill model, add the repository and that's it, but on the User model that doesn't seem to work. Is there any way I can get to manage the users as it was a normal Twill module?

Any help would be much appreciated. 

Thanks!

**Comment by antonioribeiro** [comment, marked as answer]
You can probably create a module "users" and change the User model, on this new module, to extend the App\User (App\Models\User on Laravel 8).

**Reply by manuelsofia** [reply]
Thank you for your reply Antonio. I ended up doing what you suggested but creating a module with a different name. Thanks!
